ocs_ci/ocs/perftests.py

```diff
@@ -274,26 +274,40 @@ def wait_for_wl_to_finish(self, timeout=18000, sleep=300):
         total_time = timeout
         while not Finished and total_time > 0:
             results = run_oc_command(
-                "get pod --no-headers -o custom-columns=:metadata.name,:status.phase",
+                f"get pod --no-headers -o custom-columns=:metadata.name,:status.phase",
                 namespace=benchmark_operator.BMO_NAME,
             )
-            fname = ""
+            (fname, status) = ["", ""]
             for name in results:
-                if re.search(self.client_pod_name, name):
-                    (fname, status) = name.split()
-                    continue
+                # looking for the pod which run the benchmark (not the IO)
+                # this pod contain the `client` in his name, and there is only one
+                # pod like this, other pods have the `server` in the name.
+                (fname, status) = name.split()
+                if re.search("client", fname):
```

Comment:
@Avilir should we also add `fio-` or `smallfile-` before the client so we are precisely looking for the right one? I understand we don't run fio and smallfile workload together today, but maybe we will in the future? as some sort of stress test. (This can be done in a subsequent PR as well).

Reply:
agree, i will do it in separate PR.